### PR TITLE
Feature sort filter

### PR DESCRIPTION
**Pull request by mtammineni**
Added sort and filter functionality to the getUploadStatus api

### CODE REVIEW

File: pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/loaders/UploadStatusLoader.kt [hunk 1]
```diff
@@ -30,14 +30,16 @@ class UploadStatusLoader: CosmosLoader() {
...
+    fun getUploadStatus(dataStreamId: String,
+                        dataStreamRoute: String?,
+                        dateStart: String?,
+                        dateEnd: String?,
+                        pageSize: Int,
+                        pageNumber: Int,
+                        sortBy: String?,
+                        sortOrder: String?,
+                        fileName:String?,
+                        submissionDate:String?
```
Comment (mkrystof) [inline pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/loaders/UploadStatusLoader.kt:42]: Do we need to add `submissionDate`?  We already have dateStart and dateEnd.

File: pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/loaders/UploadStatusLoader.kt [hunk 2]
```diff
@@ -57,6 +59,15 @@ class UploadStatusLoader: CosmosLoader() {
             sqlQuery.append(" and r.dataStreamRoute = '$dataStreamRoute'")
         }
 
+        fileName?.run {
+            sqlQuery.append(" and r.content.filename = '$fileName'")
+        }
+
+        submissionDate?.run {
```
Comment (mkrystof) [inline pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/loaders/UploadStatusLoader.kt:66]: I think the `submissionDate` filter can be removed - redundant.

**Review by mkrystof — CHANGES_REQUESTED** [review]

**Review by mkrystof — APPROVED** [review]

**Comment by mtammineni** [comment]
Sort and Filter for UploadStatus

**Comment by mtammineni** [comment]
Sort and Filter

**Comment by mtammineni** [comment]
Sort and Filter

**Comment by mtammineni** [comment]
Sort and filter

**Comment by mtammineni** [comment]
Sort and Filter

**Pull request merged**